code-challenges.test.js

```diff
@@ -14,50 +14,166 @@
 // Reminder: The test will call your function
 
 
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array of that length containing the numbers of the Fibonacci.
 
-// a) Create a test with expect statements for each of the variables provided.
 
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+// --------------------1) Create a function that takes in a number (greater than 2) and returns an array of that length containing the numbers of the Fibonacci.
+// a) Create a test with expect statements for each of the variables provided.
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+////////////////////////////////////////////////////////////
+
+//git a yarn jest tester template.
+// describe what I want to do
+//create a fn that can take in an array of numbers
+    //add num [0] + [1]
+    //hold that "added number" in a new array
+    // add num [2] to new, and hold it in new array
+        //new arrayy is gong to be retunred
+//input......should it be albe to take in muiltipe numbers?
+//
+
+// describe("takes in a number", () => {
+//     it("returns an fibonacci, array", () => {
+//       expect(numberGoingIn(6)).toEqual([1, 1, 2, 3, 5, 8])
+//       expect(numberGoingIn(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//     })
+// })
+
+// var fibNums = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597]
+
+
+// const numberGoingIn = (number) => {
+//      if (number >= 2){
+//         let fibOut = fibNums.slice(0, number)
+//             return fibOut
+//         }
+// }
```

Comment:
Good Attempt, not quite what's being looked for though. While what you did meets the criteria of the prompt on line 23, all of the assessment questions operate under the idea that you will solve for the desired outcomes and not call on static variables for your exact solutions. Can you think of a way that you could make this more dynamic where your function actually creates the array of fibonacci numbers?